### PR TITLE
Add OP_PICK and OP_ROLL

### PR DESCRIPTION
**Pull request by RedVelvetZip**
Added pages for OP_PICK and OP_ROLL as well as updated config to reflect their paths in the menu

### CODE REVIEW

File: opcodes/OP_PICK.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+[`OP_PICK`](./OP_PICK.md) is used to a select a stack item and copy it to the top.
+
+### Notes
+- If there are fewer than two items on the stack, if n is negative, or if n is larger than the stack when [`OP_PICK`](./OP_PICK.md) is executed, the script will fail.
+- The stack item just before OP_PICK dictates 'n', the location of the item to be copied.
+
+
+## Examples
+### Example 1
+Copying the nth stack item when n==2 with four five items (i.e., the third item from the top copies to the top):
```
Comment (thunderbiscuit) [inline opcodes/OP_PICK.md]: > with four five items

I think you mean `with five items` here maybe?

File: opcodes/OP_PICK.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# OP_PICK
+:::info
+**Opcode number:** 121
+**Byte representation:** `0x7b`
```
Comment (thunderbiscuit) [inline opcodes/OP_PICK.md]: OP_PICK is 121, and therefore 0x79

File: opcodes/OP_ROLL.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# OP_ROLL
+:::info
+**Opcode number:** 122  
+**Byte representation:** `0x7a`  
+**Short description:** Move the stack item at position n to the top of the stack.
+:::
+
+[`OP_ROLL`](./OP_ROLL.md) is used to a select a stack item and move it to the top.
```
Comment (thunderbiscuit) [inline opcodes/OP_ROLL.md]: Typo in this sentence with the extra "a".

File: opcodes/OP_ROLL.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Notes
+- If there are fewer than two items on the stack, if n is negative, or if n is larger than the stack when [`OP_ROLL`](./OP_ROLL.md) is executed, the script will fail.
+- The stack item just before OP_ROLL dictates 'n', the location of the item to be moved.
+
+
+## Examples
+### Example 1
+Moving the nth stack item when n==2 with four five items (i.e., the third item from the top moves to the top):
```
Comment (thunderbiscuit) [inline opcodes/OP_ROLL.md]: Same as above. "four five items" extra word.

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Thanks for the new pages! You need a quick rebase to pick up the latest from master, and I left a few small comments, but otherwise looks good.

**Comment by thunderbiscuit** [comment]
I think maybe a quick mention on both pages that those operations are 0-based would be good—the number 2 will reach 3 deep in the stack (or maybe it's just too obvious? Let me know what you think).

**Comment by RedVelvetZip** [comment]
thanks for the review. changes made and rebased to match the new config formatting

**Comment by thunderbiscuit** [comment]
Sorry I totally forgot to merge this! ACK e27dc16ae8a33caa6243a11885162d5d34f9a719.

I'll just do a quick cleanup of commit history and it's good to go.

**Pull request merged**